signer.js

```diff
@@ -3,12 +3,25 @@
 const cookieSignature = require('cookie-signature')
 
 module.exports = function (secret) {
+  const secrets = Array.isArray(secret) ? secret : [secret]
+  const [signingKey] = secrets
+
   return {
     sign (value) {
-      return cookieSignature.sign(value, secret)
+      return cookieSignature.sign(value, signingKey)
     },
-    unsign (value) {
-      return cookieSignature.unsign(value, secret)
+    unsign (signedValue) {
+      for (const key of secrets) {
+        const value = cookieSignature.unsign(signedValue, key)
+        if (value !== false) {
+          return {
+            renew: key !== signingKey,
+            value
+          }
+        }
+      }
+
+      return false
```

Comment:
```suggestion
      return null
```

Reply:
`unsignCookie` decorator always returns `false` for this case. So I tried to keep it same.
But it makes sense to change it to null in case of invalid. Updated.